canonical-websites/www.ubuntu.com
# Remove cloud plans and pricing for support version

## Done

* updated redirects and added one - `cloud/plans-and-pricing/?: "/support/plans-and-pricing”`
* removed the page from the navigation
* updated the link in /download/server/thank-you to support
* updated the link in the _thank_you_footer.incl
* updated the download/server/thank-you [copy doc](https://docs.google.com/a/canonical.com/document/d/1S5zY7HN_Q8eQWch0mCEfLICfj8pW1bct9_wGArCNIqc/edit?usp=drive_web)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- test that [/cloud/plans-and-pricing](http://0.0.0.0:8001/cloud/plans-and-pricing) redirects to /support/plans-and-pricing
- test that [/download/server/thank-you](http://0.0.0.0:8001/download/server/thank-you)'s Ubuntu Advantage link goes to /support/plans-and-pricing
- see that plans and pricing are not visible in the dropdown menu
